panelnet/panel_data.py

```diff
@@ -135,10 +197,10 @@ def export(idx: int, mesh, labelboxes: [LabelBox], output_folder, n_points, expo
 
     #-----pointcloud---------
     # sampled_pointcloud = mesh.sample_points_poisson_disk(number_of_points=int(n_points), init_factor=10,  use_triangle_normal=True) # better sampling
-    sampled_pointcloud = mesh.sample_points_uniformly(number_of_points=int(n_points),  use_triangle_normal=True) # quicker sampling
-
-    pc = np.asarray(sampled_pointcloud.points)
+    # sampled_pointcloud = mesh.sample_points_uniformly(number_of_points=int(n_points),  use_triangle_normal=True) # quicker sampling
 
+    # pc = np.asarray(sampled_pointcloud.points)
+    pc = np.asarray(pcd.points)
```

Comment:
Reads global pcd? pointcloud sampling should move to generate_welding_area() instead of export(), and should be returned and passed again in export()